codeChunks.Rmd

```diff
@@ -0,0 +1,156 @@
+---
+title: "Frequently Used Code Chunks"
+author: "Vivian Tran"
+date: "3/7/2018"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+These are some code chunks that I frequently come back to when processing data for the Arctic Data Center. 
+
+#Reading in raw data
+##Single data file
+```{r eval=FALSE}
+df <- read.table("path/to/data", 
+               header=T, 
```

Comment:
There should be spaces around the `=` sign.  Doesn't affect the code at all, but it makes it more readable (especially once your code gets long/complicated).  This is our go-to reference for style: http://style.tidyverse.org/